.github/workflows/on-code-change.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: Jahia/jahia-modules-action/static-analysis@v2
         with:
-          node_version: 14
+          node_version: 22
```

Comment:
just fyi, this defaults to latest lts too if you don't supply anything, and not need to maintain version.